tests/ui/rename.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(stutter)]
+#![warn(clippy::cyclomatic_complexity)]
```

Comment:
Could you make these two (+ `clippy::stutter`) `allow`? Otherwise the evaluation will stop at the first `warn` due to `-D warnings` on CI. This should result in `tests/ui/rename.stderr` including output for `clippy::stutter` again.

Reply:
Are you sure about that? [This](https://github.com/rust-lang/rust-clippy/blob/ddc718087ffeb7b932da87c26fd4e75ec6a9ac02/tests/ui/rename.stderr) is the `stderr` I get with the current file, and there Clippy is warning about stutter.

Maybe I haven't understood what you meant :thinking:

Reply:
hmm, I think I misinterpreted the stderr. It's indeed there, sorry about that =)